### PR TITLE
[Element 2.0] Support Chrome stable when running on Flood

### PR DESCRIPTION
**Pull request by phileba**

### CODE REVIEW

File: packages/core/src/driver/Playwright.ts [hunk 1]
```diff
@@ -127,9 +135,21 @@ export async function launch(
...
+		browserType = 'chromium'
+		switch (process.platform) {
+			case 'darwin':
+				options.executablePath = '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
+				break
+			case 'win32':
+				options.executablePath = 'C:/Program Files (x86)/GoogleChrome/Applicationchrome.exe'
+				break
+			default:
+				options.executablePath = '/usr/bin/google-chrome-stable'
```
Comment (honglax) [inline packages/core/src/driver/Playwright.ts:150]: What happens if the end-user install Google Chrome at their own path (not default path)?

Comment (phileba) [reply packages/core/src/driver/Playwright.ts:150]: they need to use the `executablePath`:
```
browserLaunchOptions: {
	executablePath: 'custom_path',
},
```

Comment (phileba) [reply packages/core/src/driver/Playwright.ts:150]: the purpose of this part is to support end-user who wants to run the test with the chrome stable on Flood

Comment (honglax) [reply packages/core/src/driver/Playwright.ts:150]: The executablePath of Chrome in Flood Grid is `'/usr/bin/google-chrome-stable'`? Am I correct?

**Review by HieuLeKMS — APPROVED** [review]

**Review by honglax — APPROVED** [review]
LGTM.

**Pull request merged**